rest-server/src/routes/user.js

```diff
@@ -19,21 +19,18 @@
 // module dependencies
 const express = require('express');
 const authConfig = require('../config/auth');
-const authCtrl = require('../controllers/auth');
+const authController = require('../controllers/auth');
 const param = require('../middlewares/parameter');
 
 
 const router = express.Router();
 
 router.route('/')
-    /** POST /api/auth - Return token if username and password is correct */
-    .post(param.validate(authConfig.schema), authCtrl.login)
+    /** PUT /api/v1/user - Create or update a user */
+    .put(authConfig.check, param.validate(authConfig.schema), authController.updateUser)
 
-    /** PUT /api/auth - Update user */
-    .put(authConfig.check, param.validate(authConfig.schema), authCtrl.update)
-
-    /** DELETE /api/auth - Delete user */
-    .delete(authConfig.check, authCtrl.remove);
+    /** DELETE /api/v1/user - Remove a user */
+    .delete(authConfig.check, authController.removeUser);
```

Comment:
if you change `auth` route, `auth` controller and model should be changed correspondingly.

Reply:
ok. but ‘user’ and ‘token’ share the same underlying model to access lowdb and to encrypt passwords. do you suggest we should split the model into two also?